src/main/java/org/mule/service/scheduler/internal/ThrottledScheduler.java

```diff
@@ -59,7 +59,7 @@ protected void putTask(RunnableFuture<?> task, ScheduledFuture<?> scheduledFutur
     try {
       synchronized (runningTasks) {
         runningTasks.incrementAndGet();
-        while (runningTasks.get() >= maxConcurrentTasks) {
+        while (runningTasks.get() > maxConcurrentTasks) {
```

Comment:
no tests for this?

Reply:
That would be the one in file but let me see if I can create a test case within the scheduler service.